addons/mail/static/src/core/common/store_service.js

```diff
@@ -766,7 +766,7 @@ export class Store extends BaseStore {
             ...thread.getFetchParams(),
             fetch_params: {
                 is_notification,
-                search_term: await prettifyMessageText(searchTerm), // formatted like message_post
+                search_term: await prettifyMessageText(searchTerm, thread), // formatted like message_post
```

Comment:
From previous comment, the `thread` should be optional named param in 2nd positional. Currently it's implemented as 3rd param thus this diff is wrong.